training_record/services.py

```diff
@@ -269,6 +270,10 @@ def create_campus_records_from_excel(file, context):
                 PermissionService.assign_object_permissions(user, record)
                 records.add(record)
 
+                msg = (f'管理员{admin}创建了用户{record.user}参加'
```

Comment:
Use `{user}` instead, I'm not sure whether `record.user` is already retrieved from database or not, but `{user}` is definitely already here.

Reply:
BTW, I think you can add another `set()` to track users, we don't want to create multiple records for single user in one batch submit.